desktop/webpack.config.dll-build.js

```diff
@@ -0,0 +1,25 @@
+var path = require('path')
+var webpack = require('webpack')
+
+module.exports = {
+  entry: {
+    vendor: ['material-ui', 'react', 'lodash', 'immutable', 'material-ui/styles', 'material-ui/svg-icons', 'material-ui/FlatButton', 'material-ui/Popover', 'moment', 'lodash', 'html-entities', 'redux-saga', 'core-js', 'qrcode-generator', 'react-json-tree', 'redux'],
```

Comment:
`lodash` is listed twice in this list. Perhaps sort it to make it easier to catch dupes?